_config.yml

```diff
@@ -11,6 +11,7 @@ github_username:  opensourcecatholic
 mailinglist_name: open-source-catholic
 permalink: /:year/:title
 paginate: 7
+post_preview_words: 42
```

Comment:
I'd say maybe double this, just so its easier to get a good idea of the contents on the first glance.